index.js

```diff
@@ -14,4 +14,128 @@ cloSerX.addEventListener('click', () => {
 navListItems.forEach((item) => item.addEventListener('click', () => {
   mariMarzi.classList.add('hide');
   mariMarzi.classList.remove('mobile-menu');
-}));
\ No newline at end of file
+}));
+function createPopupWindow(project) {
+  const popupWindow = document.createElement('div');
+  popupWindow.classList.add('popup-window');
+  popupWindow.innerHTML = `<div>
+      <a id="close-popup"><img src="files/Icon.svg" alt="Close Button"></a>
+      <h2 class="card-header">${project.name}</h2>
+      <ul class="card-badges"></ul>`;
+  project.techs.forEach((item) => {
+    popupWindow.querySelector('ul').innerHTML += `<li>${item}</li>`;
+  });
+  popupWindow.querySelector('div').innerHTML += `
+      <div class="img-placeholder"><img src="${project.imgSrc}" alt="The project preview"></div>
```

Comment:
-  [ ]  Project images in your modal windows are not displaying for some of your modal windows
![img](https://user-images.githubusercontent.com/99656687/233049024-07408ec0-3553-47be-8b89-99679eae16d1.png)
**Please fix this and test your changes both locally and on [githack](https://raw.githack.com/)**

Reply:
Hello @samiullah12345678, Kindly note that I errored and submitted the reviews before saving some of the comments. I have therefore added them and saved them again. This will make it appear as 2 different reviews. Please work with the knowledge that it is one review. Thanks

Reply:
Also, note that it is not a good idea to keep working on your project once it si submitted for review. This can lead to inconsistent reviews which may end up confusing the student